tests/integration/monitoring/test-monitoring.go

```diff
@@ -163,6 +170,15 @@ func testTargetsAreHealthy() {
 
 }
 
+func hasLabels(targetLabels, labelsToBeIgnored promAPI.Labels) bool {
```

Comment:
(nit) Wouldn't `hasAnyLabel` be a better name 🤔, because the function does not check if all `labelsToBeIgnored ` are part of `targetLabels`. Instead it returns, when one key/value pair from `labelsToBeIgnored` is found in `targetLabels`. 
```suggestion
func hasAnyLabel(targetLabels, labelsToBeIgnored promAPI.Labels) bool {
```

Reply:
(nit) i think the name is correct, just the code is the wrong way around....
loop over labels to be ignored and check if the target has those ... not the other way round

Reply:
changed to `hasAnyLabel `